rpc/contract.go

```diff
@@ -55,7 +55,7 @@ func (provider *Provider) ClassAt(ctx context.Context, blockID BlockID, contract
 // Returns:
 // - ClassOutput: a ClassOutput interface
 // - error: an error if any
-func typecastClassOutput(rawClass *map[string]any) (ClassOutput, *RPCError) {
+func typecastClassOutput(rawClass *map[string]any) (ClassOutput, error) {
```

Comment:
Just curious why it's ptr to map? (default value of map is nil, could it work without ptr? )

Reply:
Can't remember, maybe I thought passing by reference would be more efficient. Updated it to use a non-ptr